packages/design-system/src/styles/components/_SingleInputDateField.scss

```diff
@@ -397,6 +397,7 @@
     border-bottom-left-radius: 0;
     color: $color-primary;
     cursor: pointer;
+    margin: 0;
```

Comment:
I can't leave a comment because the change is not part of this PR, but on line 368, I see a warning that says
> start value has mixed support, consider using flex-start instead